lasagne/utils.py

```diff
@@ -236,11 +259,11 @@ def create_param(spec, shape, name=None):
 
     Parameters
     ----------
-    spec : numpy array, Theano shared variable, or callable
+    spec : numpy array, Theano expression, or callable
         Either of the following:
 
         * a numpy array with the initial parameter values
-        * a Theano shared variable representing the parameters
+        * a Theano expression representing the parameters
```

Comment:
I think that's a good idea, since that is the most common use case. Maybe something like "usually this is a Theano shared variable."